src/opds_dumper.cpp

```diff
@@ -95,6 +92,25 @@ BookData getBookData(const Library* library, const std::vector<std::string>& boo
       {"url", bookUrl},
       {"size", to_string(book.getSize())},
       {"icons", getBookIllustrationInfo(book)},
+    };
+}
+
+std::string getSingleBookEntryXML(const Book& book, bool withXMLHeader, const std::string& endpointRoot, bool partial)
+{
+  auto data = getSingleBookData(book);
+  data["with_xml_header"] = MustacheData(withXMLHeader);
+  data["dump_partial_entries"] = MustacheData(partial);
+  data["endpoint_root"] = endpointRoot;
+  return render_template(RESOURCE::templates::catalog_v2_entry_xml, data);
+}
+
+BookData getBookData(const Library* library, const std::vector<std::string>& bookIds, const std::string& endpointRoot, bool partial)
```

Comment:
Make this plural ? (`BooksData getBooksData(...)`)

Reply:
Done